docker_xylem/tests/test_docker_xylem.py

```diff
@@ -103,3 +104,11 @@ def test_list(self):
 
         self.assertEquals(result['Err'], None)
         self.assertEquals(result['Volumes'][0]['Name'], 'testvol')
+
+    @defer.inlineCallbacks
+    def test_capabilities(self):
+
+        result = yield self.service._route_request(FakeRequest(
```

Comment:
All test methods should have a docstring that explains what the test is supposed to be testing. The test name is too short to carry that information and the test code "explains" what the test is *actually* testing, which might not be the same as what it's *supposed* to be testing. (Like all code, tests can have bugs in them.)

This isn't a thing the existing code has, so don't worry about adding docstring to other tests -- only do it for the new tests you write, including this one.

A suitable docstring for this particular test might be:
```python
def test_capabilities(self):
    """
    VolumeDriver.Capabilities always returns {'Scope': 'global'}.
    """
    result = ...
```